USERS.md

```diff
@@ -11,126 +11,127 @@ As the Argo Community grows, we'd like to keep track of our users. Please send a
-1. [Helio](https://helio.exchange)
-1. [HOVER](https://hover.to)
-1. [HSBC](https://hsbc.com)
-1. [IBM](https://ibm.com)
-1. [InsideBoard](https://www.insideboard.com)
-1. [Interline Technologies](https://www.interline.io/blog/scaling-openstreetmap-data-workflows/)
-1. [Intralinks](https://www.intralinks.com/)
-1. [Intuit](https://www.intuit.com/)
-1. [InVision](https://www.invisionapp.com/)
-1. [İşbank](https://www.isbank.com.tr/en)
-1. [Jungle](https://www.jungle.ai/)
-1. [Karius](https://www.kariusdx.com/)
-1. [KarrotPay](https://www.daangnpay.com/)
-1. [Kasa](https://www.kasa.co.kr/)
-1. [KintoHub](https://www.kintohub.com/)
-1. [Localytics](https://www.localytics.com/)
-1. [Maersk](https://www.maersk.com/solutions/digital-solutions)
-1. [Max Kelsen](https://maxkelsen.com/)
-1. [Microba](https://www.microba.com/)
-1. [Microblink](https://microblink.com/)
-1. [Mirantis](https://mirantis.com/)
-1. [Mixpanel](https://mixpanel.com)
-1. [Motus](https://www.motus.com)
-1. [New Relic](https://newrelic.com/)
-1. [Nikkei](https://www.nikkei.co.jp/nikkeiinfo/en/)
-1. [Norwegian Refugee Council](https://www.nrc.no/)
-1. [nrd.io](https://nrd.io/)
-1. [NVIDIA](https://www.nvidia.com/)
-1. [Onepanel](https://docs.onepanel.ai)
-1. [OVH](https://www.ovh.com/)
-1. [Peak AI](https://www.peak.ai/)
-1. [PDOK](https://www.pdok.nl/)
-1. [Polarpoint.io](https://polarpoint.io)
-1. [Pollination](https://pollination.cloud)
-1. [Preferred Networks](https://www.preferred-networks.jp/en/)
-1. [Prudential](https://www.prudential.com.sg/)
-1. [Quantibio](http://quantibio.com/us/en/)
-1. [Raccoon Digital Marketing](https://raccoon.ag/)
-1. [Ramboll Shair](https://ramboll-shair.com/)
-1. [Ravelin](https://www.ravelin.com/)
-1. [Red Hat](https://www.redhat.com/en)
-1. [Reserved AI](https://reserved.ai/)
-1. [Riskified](https://www.riskified.com)
-1. [Robinhood](https://robinhood.com/)
-1. [Sage (Sage AI Labs)](https://sage.com/)
-1. [SAP Fieldglass](https://www.fieldglass.com/)
-1. [SAP Hybris](https://cx.sap.com/)
-1. [SAS](https://www.sas.com/)
-1. [Schlumberger](https://slb.com/)
-1. [SegmentStream](https://segmentstream.com)
-1. [Sendible](https://sendible.com)
-1. [Sidecar Technologies](https://hello.getsidecar.com/)
-1. [Softonic](https://hello.softonic.com/)
-1. [Sohu](https://www.sohu.com/)
-1. [Stillwater Supercomputing, Inc](http://www.stillwater-sc.com/)
-1. [strongDM](https://www.strongdm.com/)
-1. [Styra](https://www.styra.com/)
-1. [Threekit](https://www.threekit.com/)
-1. [Tiger Analytics](https://www.tigeranalytics.com/)
-1. [Tulip](https://tulip.com/)
-1. [UFirstGroup](https://www.ufirstgroup.com)
-1. [Vispera](https://www.vispera.co)
-1. [Wavefront](https://www.wavefront.com/)
-1. [Wellcome Trust](https://wellcome.ac.uk/)
-1. [VMware](https://www.vmware.com/)
-1. [WooliesX](https://wooliesx.com.au/)
-1. [Woolworths Group](https://www.woolworthsgroup.com.au/)
-1. [Zhihu](https://www.zhihu.com/)
+2. [4intelligence](https://www.4intelligence.com.br/)
```

Comment:
I think the repo uses `1.` for the ease of maintenance.
these will automatically render an ordered list, no need to specify if this is `2.` or `3.`
Could you change these back to `1.` please